hack/schema_validation.sh

```diff
@@ -1,28 +1,115 @@
 #!/bin/bash
+###############################################################################
+#                                                                             #
+#      Check Managed Scripts metadata.yaml files against the JSON Schema      #
+#                                                                             #
+###############################################################################
+#
+# release: v2.0
+# description: Script responsible for listing the metadata.yaml files in the
+#              managed scripts directory and for running the validation 
+#              against the Json Schema.
+# team: SREP
+# author: Fabio Aldana
+# email: faldana@redhat.com
+#
 
 #set -ex
 
-cd $(dirname $0)/..
+# define variables
 CONTAINER_ENGINE=$(which docker 2>/dev/null || which podman)
-#convert all metadata.yaml to json first
-yfiles=$(find . -name 'metadata.yaml')
-for f in $yfiles
+CONTAINER_PATH=/json
+
+# change to the managed-scripts directory on your repository
+cd $(dirname $0)/..
+
+# funtion to present the help
+usage() {
```

Comment:
Can we set the intents for the scripts inside the function for easy reading?

Reply:
Done.